src/migrations/20230709103710-deleted-phone-to-array.ts

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  async up(db) {
+    console.log("Begin migration for deletedPhone")
+
+    const users = await db.collection("users").find({}).toArray()
+    for (const user of users) {
+      if (user.deletedPhone) {
+        // If deletedPhone exists and it's not an array, convert it to an array
+        if (!Array.isArray(user.deletedPhone)) {
```

Comment:
Does this have to be done pro-actively? Can't it be done lazily as users update their phone numbers?

Reply:
doesn't have to, but it make things less complex going forward. https://github.com/GaloyMoney/galoy/pull/2873#discussion_r1257447221